### PR TITLE
PHP8.4 Support Added

### PR DESCRIPTION
**Pull request by cod43156**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Review by gsteel — CHANGES_REQUESTED** [review]
You can ignore platform requirements for 8.4 to get dependencies installed like in this PR: https://github.com/laminas/laminas-mvc-i18n/pull/41/files

You may need to bump `laminas/stdlib` to `3.19.0` minimum for 8.4 compat and you will likely need to bump dev deps with `composer bump -D`

Please also sign off your commits as per the [contributing guide](https://github.com/laminas/.github/blob/main/CONTRIBUTING.md#commit-signoffs)

**Comment by cod43156** [comment]
> You can ignore platform requirements for 8.4 to get dependencies installed like in this PR: https://github.com/laminas/laminas-mvc-i18n/pull/41/files
> 
> You may need to bump `laminas/stdlib` to `3.19.0` minimum for 8.4 compat and you will likely need to bump dev deps with `composer bump -D`
> 
> Please also sign off your commits as per the [contributing guide](https://github.com/laminas/.github/blob/main/CONTRIBUTING.md#commit-signoffs)

@gsteel The comment has been resolved. I' ve bumped laminas/stdlib to ^3.19.0 for 8.4 compatibility, and ignored platform requirements.

**Comment by gsteel** [comment]
Please can you also fix the minor docs linting issues so that we have a green build?

**Comment by cod43156** [comment]
> Please can you also fix the minor docs linting issues so that we have a green build?
@gsteel lint issues fixed

File: .laminas-ci.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.1": true,
+        "8.4": true
```
Comment (samsonasik) [inline .laminas-ci.json]: can this only php 8.4?

Comment (cod43156) [reply .laminas-ci.json]: 8.1 removed

**Review by samsonasik — APPROVED** [review]

**Review by gsteel — APPROVED** [review]
Thanks @cod43156

**Pull request merged**